gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/formatter/GherkinPrettyFormatter.java

```diff
@@ -0,0 +1,101 @@
+package gherkin.utils.formatter;
```

Comment:
The package should be `io.cucumber.gherkin.utils`.

Reply:
Thank you, I have already changed package name to ```io.cucumber.gherkin.utils```